core/math/math_2d.h

```diff
@@ -43,6 +43,14 @@ enum Margin {
 	MARGIN_BOTTOM
 };
 
+enum Corner {
```

Comment:
If it's used only for StyleBoxFlat, should it really go in math_2d.h or could it be kept in style_box_flat.h?